web/src/components/shared/cart/cart.tsx

```diff
@@ -17,12 +18,13 @@ const FaqWrapper = styled.div`
 `;
```

Comment:
instead of doing it like this it is better to leave this component as it should be, with the real call to get context and everything and make getCart return the mock